services/web/server/src/simcore_service_webserver/folders/_folders_db.py

```diff
@@ -80,14 +93,16 @@ async def list_(
     user_id: UserID | None,
     workspace_id: WorkspaceID | None,
     product_name: ProductName,
+    trashed: bool | None,
     offset: NonNegativeInt,
     limit: int,
     order_by: OrderBy,
 ) -> tuple[int, list[FolderDB]]:
     """
     content_of_folder_id - Used to filter in which folder we want to list folders. None means root folder.
+    trashed - If set to true, it returns folders **explicitly** trashed, if false then non-trashed folders.
```

Comment:
so as in not the ones implicitely trashed? I don't quite get it

Reply:
I hope the explanations above are clarifying, otherwise lets talk offline.

Reply:
clarified offline.